Assignment_web/project1/application.py

```diff
@@ -87,10 +87,7 @@ def authenticate():
         # return render_template("print.html",first=name,EMAIL = email)
         return redirect(url_for('index'))
     except Exception :
-	    return render_template("error.html", errors = "Details are already given")   
```

Comment:
Can you explain the use of Registrations.query.all()?You are not storing  password while registration which is a mandatory field.